### DISCUSSION TITLE
Current swagger.json is not structurally or semantically correct

### DISCUSSION THREAD

**Discussion by AryanJ-NYC** [btcpayserver/btcpayserver#3384] [Ideas / Feature Requests]
I maintain a node.js BTCPay Greenfield client. I generate the functions from https://docs.btcpayserver.org/API/Greenfield/v1/swagger.json.

It seems the `swagger.json` is not structurally or semantically correct.

# Steps to Recreate
1. Go to https://editor.swagger.io/
2. Click File | Import from URL
3. Enter https://docs.btcpayserver.org/API/Greenfield/v1/swagger.json

You'll see something like:
<img width="572" alt="Screen Shot 2022-01-28 at 9 29 11 PM" src="https://user-images.githubusercontent.com/8263430/151643840-8f455a7a-3f39-44a6-9fc7-922dad4b5d94.png">

It'd be ultra-helpful to have the Swagger be correct so that generators can easily create clients for them.

**Comment by bolatovumar** [comment]
Thanks for reporting @AryanJ-NYC! See the PR which addresses this here: https://github.com/btcpayserver/btcpayserver/pull/3390
